main/adapters/repository/init_postgres.go

```diff
@@ -27,28 +28,33 @@ type Migration struct {
 }
 
 var CREATE = map[int]string{
-	PostgresIdentity: "CREATE TABLE IF NOT EXISTS " + vars.PostgreSqlIdentityTableName + "(" +
+	PostgresIdentity: "CREATE TABLE IF NOT EXISTS " + vars.SqlIdentityTableName + "(" +
 		"uid VARCHAR(255) NOT NULL PRIMARY KEY, " +
 		"private_key BYTEA NOT NULL, " +
 		"public_key BYTEA NOT NULL, " +
 		"signature BYTEA NOT NULL, " +
 		"auth_token VARCHAR(255) NOT NULL);",
-	PostgresVersion: "CREATE TABLE IF NOT EXISTS " + vars.PostgreSqlVersionTableName + "(" +
+	PostgresVersion: "CREATE TABLE IF NOT EXISTS " + vars.SqlVersionTableName + "(" +
 		"id VARCHAR(255) NOT NULL PRIMARY KEY, " +
 		"migration_version VARCHAR(255) NOT NULL);",
-	//MySQL:    "CREATE TABLE identity (id INT, datetime TIMESTAMP)",
-	//SQLite:   "CREATE TABLE identity (id INTEGER, datetime TEXT)",
-}
 
-type Table struct {
-	exists bool
+	SQLiteIdentity: "CREATE TABLE IF NOT EXISTS " + vars.SqlIdentityTableName + "(" +
```

Comment:
is this supposed to be in file init_postgres.go?

Reply:
i ve moved it out should be better now